### PR TITLE
Issue 2312 - Autowire FirestoreMappingContext into FirestoreTemplate

### PR DESCRIPTION
**Pull request by acet**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@acet Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-gcp&pullRequestId=2313)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-gcp&pullRequestId=2313) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by meltsufin** [comment]
@acet Thanks for the contribution. Please make sure that you've signed the CLA so that we can merge this in.

**Comment by pivotal-issuemaster** [comment]
@acet Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-gcp&pullRequestId=2313)!

File: spring-cloud-gcp-data-firestore/src/test/java/org/springframework/cloud/gcp/data/firestore/it/FirestoreIntegrationTestsConfiguration.java [hunk 1]
```diff
@@ -65,7 +65,7 @@
 	@Bean
 	public FirestoreTemplate firestoreTemplate(FirestoreGrpc.FirestoreStub firestoreStub,
 			FirestoreClassMapper classMapper) {
-		return new FirestoreTemplate(firestoreStub, this.defaultParent, classMapper);
+		return new FirestoreTemplate(firestoreStub, this.defaultParent, classMapper, new FirestoreMappingContext());
```
Comment (meltsufin) [inline spring-cloud-gcp-data-firestore/src/test/java/org/springframework/cloud/gcp/data/firestore/it/FirestoreIntegrationTestsConfiguration.java:68]: `FirestoreMappingContext` bean already exists in the configuration class. Simply wire it into the method.

**Pull request merged**